user/languages/javascript-with-nodejs.md

```diff
@@ -134,6 +134,20 @@ For more information, refer to [Caching](/user/caching) documentation.
 
 Note that `npm install` can fail if a shrinkwrapped git dependency pointing to a branch has its HEAD changed.
 
+### Using specific version of `npm` or `yarn`
+
+Add the following to the `before_install` phase of `.travis.yml`:
+
+```yaml
+before_install:
+  # for npm
+  - npm i -g npm@version-number
+  
```

Comment:
I've changed it as shown in my last comment https://github.com/travis-ci/docs-travis-ci-com/pull/1290#issuecomment-312686644 (& waiting approval from @plaindocs to push them) but I'm starting to think about rewording:
> Add the following to the before_install phase of .travis.yml:

to maybe:
> Add the following to the before_install phase of the build inside your .travis.yml:

and in your [using gulp's doc](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Using-Gulp), you just ask to add the lines to travis without saying anything about build stages:

> If you already use Gulp to manage your tests, install it and run the default gulpfile.js by **adding the following lines to your .travis.yml**:

Should I do the same or this is fine... not sure!